pipelines/wdl/arrays/imputation_beagle/input_qc/test_inputs/Plumbing/NA12878_chr20_qc_succeeds.json

```diff
@@ -0,0 +1,8 @@
+{
+    "ArrayImputationQC.multi_sample_vcf": "gs://broad-gotc-test-storage/imputation_beagle/input_qc/scientific/vcfs/test_qc_only_chr20.vcf.gz",
+    "ArrayImputationQC.ref_dict": "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.dict",
+    "ArrayImputationQC.reference_panel_path_prefix": "gs://broad-gotc-test-storage/imputation_beagle/scientific/1000G_HGDP_no_singletons_reference_panel/hgdp.tgp.gwaspy.AN_added.bcf.ac2",
+    "ArrayImputationQC.contigs": ["chr20","chr21","chr22"],
```

Comment:
this succeeds because chr20 is in the file and chr20 is in this list right? (just making sure i understand)

Reply:
nah this is fine was just double checking